src/gretel_client/client.py

```diff
@@ -527,9 +527,22 @@ def get_project(
             return self._create_get_project()
 
     def install_transformers(self):
+        """Deprecated: Installs the latest version of the Gretel Transformers package
+
+        Prefer ``install_packages`` instead.
+        """
+        logger.warn(
```

Comment:
nit: I think `logger.warning` is the preferred as `warn` is deprecated since py 3.3 or something